src/openforms/emails/tests/test_tasks_integration.py

```diff
@@ -0,0 +1,209 @@
+
+from ..tasks import send_email_digest
+
+
+@override_settings(LANGUAGE_CODE="en")
+class EmailDigestTaskIntegrationTests(TestCase):
+
+    def test_that_repeated_failures_are_not_mentioned_multiple_times(self):
+        submission = SubmissionFactory.create()
+
+        with freeze_time("2023-01-02T12:30:00+01:00"):
+            logevent.email_status_change(
+                submission,
+                event="registration",
+                status=Message.STATUS_FAILED,
+                status_label="Failed",
+                include_in_daily_digest=True,
+            )
+        with freeze_time("2023-01-02T13:30:00+01:00"):
+            logevent.email_status_change(
+                submission,
+                event="registration",
+                status=Message.STATUS_FAILED,
+                status_label="Failed",
+                include_in_daily_digest=True,
+            )
+
+        with (
+            freeze_time("2023-01-03T01:00:00+01:00"),
+            patch(
+                "openforms.emails.tasks.GlobalConfiguration.get_solo",
+                return_value=GlobalConfiguration(
+                    recipients_email_digest=["tralala@test.nl", "trblblb@test.nl"]
+                ),
+            ),
+        ):
+            send_email_digest()
+
+        sent_email = mail.outbox[0]
+        submission_occurencies = sent_email.body.count(str(submission.uuid))
+
+        self.assertEqual(
+            sent_email.subject, "[Open Forms] Daily summary of detected problems"
+        )
+        self.assertEqual(
+            sent_email.recipients(), ["tralala@test.nl", "trblblb@test.nl"]
+        )
+        self.assertEqual(submission_occurencies, 1)
+
+    def test_no_email_sent_if_no_logs(self):
+        with patch(
+            "openforms.emails.tasks.GlobalConfiguration.get_solo",
+            return_value=GlobalConfiguration(
+                recipients_email_digest=["tralala@test.nl"]
+            ),
+        ):
+            send_email_digest()
+
+        self.assertEqual(0, len(mail.outbox))
+
+    def test_no_email_sent_if_no_recipients(self):
+        submission = SubmissionFactory.create()
+
+        with freeze_time("2023-01-02T12:30:00+01:00"):
+            logevent.email_status_change(
+                submission,
+                event="registration",
+                status=Message.STATUS_FAILED,
+                status_label="Failed",
+                include_in_daily_digest=True,
+            )
+
+        with (
+            freeze_time("2023-01-03T01:00:00+01:00"),
+            patch(
+                "openforms.emails.tasks.GlobalConfiguration.get_solo",
+                return_value=GlobalConfiguration(recipients_email_digest=[]),
+            ),
+        ):
+            send_email_digest()
+
+        self.assertEqual(0, len(mail.outbox))
+
+    @patch(
+        "openforms.emails.tasks.GlobalConfiguration.get_solo",
+        return_value=GlobalConfiguration(recipients_email_digest=["tralala@test.nl"]),
+    )
```

Comment:
I think this can be moved to the test case class as a decorator, since it applies for all tests. For the `test_no_email_sent_if_no_recipients` test, you can then do:

```py
def test_no_email_sent_if_no_recipients(self, mock_global_config):
    mock_global_config.return_value.recipients_email_digest = []
```